src/main/java/com/tstepnik/planner/domain/task/TaskMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.tstepnik.planner.domain.task;
+
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface TaskMapper {
+
+    TaskDTO toDto(Task task);
+
+    Task toTask(TaskDTO taskDto);
```

Comment:
```suggestion
    Task toEntity(TaskDTO taskDto);
```

Reply:
Does toEntity isn't confusing? User is entity as well.